bundles/net.tourbook.tests/test/utils/Comparison.java

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+ * Copyright (C) 2020 Frédéric Bard
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation version 2 of the License.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110, USA
+ *******************************************************************************/
+package utils;
+
+import de.byteholder.geoclipse.map.UI;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+import net.tourbook.common.util.StringUtils;
+import net.tourbook.data.TourData;
+
+import org.skyscreamer.jsonassert.ArrayValueMatcher;
+import org.skyscreamer.jsonassert.Customization;
+import org.skyscreamer.jsonassert.JSONCompare;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+import org.skyscreamer.jsonassert.JSONCompareResult;
+import org.skyscreamer.jsonassert.comparator.CustomComparator;
+
+public class Comparison {
+
+   private static final String JSON = ".json"; //$NON-NLS-1$
+
+   /**
+    * Compares a test transaction against a control transaction.
+    *
+    * @param testTourData
+    *           The generated test TourData object.
+    * @param controlFileName
+    *           The control's file name.
+    */
+   public static void CompareTourDataAgainstControl(final TourData testTourData,
```

Comment:
Method names should always start with lowercase, Uppercase is used for classnames

Reply:
Fixed in my [PR](https://github.com/wolfgang-ch/mytourbook/pull/255)